.github/workflows/conformance.yml

```diff
@@ -62,38 +68,34 @@ jobs:
           ref: "conformance/v0.5.0-skipped-tests"
           path: "gateway-api"
 
-      - name: Setup Goenv
-        uses: ./consul-api-gateway/.github/actions/goenv
+      - name: Setup Kind cluster
+        if: matrix.cluster-type == 'kind'
+        uses: ./consul-api-gateway/.github/actions/setup-kind
         with:
-          go-version: ${{ env.GO_VERSION }}
-
-      - name: Build binary
-        env:
-          CGO_ENABLED: "0"
-          GOARCH: "amd64"
-          GOOS: "linux"
-        working-directory: "consul-api-gateway"
-        run: go build -o ./consul-api-gateway
+          metallb-config-path: "./consul-api-gateway/internal/testing/conformance/metallb-config.yaml"
 
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+      - name: Generate Unique Cluster Name
+        if: matrix.cluster-type == 'eks'
+        id: unique
+        run:
+            echo "::set-output name=clustername::$( echo consul-api-gateway-test-${{ github.run_number }}-$RANDOM )"
 
-      # docker build -f Dockerfile.local -t consul-api-gateway:test --platform linux/amd64 .
-      - name: Build Docker image
-        uses: docker/build-push-action@v2
+      - name: Configure AWS credentials
+        if: matrix.cluster-type == 'eks'
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
Instead of having a separate step here and having to keep adding more similar steps here in the main workflow for GKE + AKS, is it possible to just pass the values below into the `setup-eks` action and run this step there?

Reply:
I'll test